### PR TITLE
Update hook.py

### PR DESCRIPTION
**Pull request by vishalsubbiah**
Based on comment here: https://github.com/OpenMined/PySyft/pull/2913#discussion_r367878631 
`HookedTensor` is missing a `numpy_tensor` argument.
Submitting the fix.

### CODE REVIEW

**Comment by karlhigley** [comment]
Thanks for the fix! Looks like the build is saying there’s a code formatting issue of some kind. Set up the [commit hooks](https://github.com/OpenMined/PySyft/blob/master/CONTRIBUTING.md) and they’ll format the code for you when make a commit.

**Comment by vishalsubbiah** [comment]
@karlhigley I fixed the formatting in the second commit. The build is still running for it.  Thanks for the link to the commit hooks, that will help in the future.

File: syft/frameworks/torch/tensors/interpreters/hook.py [hunk 1]
```diff
@@ -8,7 +8,9 @@ class HookedTensor(AbstractTensor):
     method hooked without having to add it to the hook.py file.
     """
 
-    def __init__(self, owner=None, id=None, tags=None, description=None, verbose=False):
+    def __init__(
+        self, numpy_tensor=None, owner=None, id=None, tags=None, description=None, verbose=False
```
Comment (karlhigley) [inline syft/frameworks/torch/tensors/interpreters/hook.py]: I think we actually want to make the reverse change here; the Numpy code got moved over to [NumpyTensor](https://github.com/OpenMined/PySyft/blob/be9e0d894e7fe09cf22f93a3a27ad42a3903a037/syft/frameworks/torch/tensors/interpreters/numpy.py), so we should probably remove both the `numpy_tensor` parameter and the docstring explanation of it.

Comment (vishalsubbiah) [reply syft/frameworks/torch/tensors/interpreters/hook.py]: oh okay. fixed it now.

**Comment by karlhigley** [comment]
@vishalsubbiah Thanks for the PR!

**Pull request merged**